src/prefect/utilities/agent.py

```diff
@@ -1,3 +1,5 @@
+from distutils.version import LooseVersion
```

Comment:
Ah good call using a built-in!

Reply:
Yeah I hate welcoming in extra dependencies